include/ghex/communication_object_2.hpp

```diff
@@ -47,7 +47,7 @@ namespace gridtools {
             using co_t              = communication_object<Communicator,GridType,DomainIdType>;
             using communicator_type = Communicator;
 
-        private: // members
+        public: // members
```

Comment:
Why do they need to be public? For testing? I'm using somewhere the trick of friending to a test class to be implemented in the tests...

Reply:
I needed this to get access to std::function - I will change it